src/pages/Promo/Promo.js

```diff
@@ -0,0 +1,125 @@
+import React from "react";
+import { View, Text, Button, Alert,FlatList,Image } from "react-native";
+import Background from "../../components/Background/Background";
+import {StyleSheet} from 'react-native';
+
+
+const Promo = () => {
+  const Items = [
```

Comment:
Dummy data key name is better if refer to the db schema
https://github.com/Five-Fishes/On-Demand-Car-Servicing-Backend/blob/dev/docs/schema.json